src/in/shick/diode/markdown/Markdown.java

```diff
@@ -79,7 +79,7 @@
     static final RunAutomaton autoLinkUrlAutomaton = new RunAutomaton(new RegExp("((https?|ftp):([^'\"> \t\r\n])+)", RegExp.NONE).toAutomaton());
 //    static final Pattern autoLinkEmail = Pattern.compile("<([-.\\w]+\\@[-a-z0-9]+(\\.[-a-z0-9]+)*\\.[a-z]+)>");
 
-    static final RunAutomaton subredditAutomaton = new RunAutomaton(new RegExp("/[rR]/[a-zA-Z0-9]+/?", RegExp.NONE).toAutomaton());
+    static final RunAutomaton subredditAutomaton = new RunAutomaton(new RegExp("[rR]/[a-zA-Z0-9]+/?", RegExp.NONE).toAutomaton());
```

Comment:
I was able to partially solve it with the expression "[ /][rR]/[a-zA-Z0-9]+/?" : As long as there is a space preceding or a slash preceding the r/, it will be recognized as a subreddit. The problem is that if the line starts with r/subname, it won't be recognized (because there is no / or space). I've been looking through the docs and it looks like /w or /b should work, but they don't. Any ideas?